File: c_implementation/interactive_prompt.c
#include <stdio.h>

static char input[2048];

int main(int argc, char** argv){
  puts("Anotlisp version 0.0.0.1");
  puts("Press Ctrl+c to Exit \n");

    while (1){
      fputs("Anotlisp> ", stdout);

      fgets(input, 2048, stdin);

      printf("No you're a %s", input);
  }
  return 0;
}

File: rust_implementation/src/interactive_prompt.rs
use std::io;
use std::io::Write;

fn main() -> io::Result<()> {
    let mut buffer = String::new();

    println!("Anotlisp version 0.0.0.1");
    println!("Press Ctrl+C to Exit \n");

    loop {
        print!("Anotlisp> ");
        io::stdout().flush()?;
        io::stdin().read_line(&mut buffer)?;

        println!("No you're a {}", buffer.trim());
        buffer.clear();
    }
}
